ApollosProject/apollos-prototype
# Move MapView component to package

## DESCRIPTION
<img src="https://user-images.githubusercontent.com/2528817/68688455-60e0b980-0534-11ea-857f-48ca800cad98.png" width="200">

### What does this PR do, or why is it needed?

Moves `MapView` component to its own package. This continues with the team goal to continue to componentize the Apollos project.

### How do I test this PR?

* Open a simulator and go to user profile
* Click the gear icon to go to user settings
* Click Location
* Play around with the map and selecting your campus and make sure everything works as expected.

https://v.usetapes.com/JvL3NbSzi8
https://v.usetapes.com/2x63jjdxNW

### What automated tests did you write?

Tested with Snapshot test.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
